src/entity/routes/entityRouter.ts

```diff
@@ -1,13 +1,12 @@
 import { Router } from 'express';
 import { FactoryFunction } from 'tsyringe';
-import { validate } from 'openapi-validator-middleware';
 import { EntityController } from '../controllers/entityController';
 
 const entityRouterFactory: FactoryFunction<Router> = (dependencyContainer) => {
   const router = Router();
   const controller = dependencyContainer.resolve(EntityController);
 
-  router.get('/', validate, controller.get.bind(controller));
+  router.post('/', controller.post.bind(controller));
```

Comment:
The controller uses arrow functions, so can can remove `.bind`

Reply:
done